app/javascript/components/upload/UploadWizard.jsx

```diff
@@ -35,6 +35,9 @@ import MiscellaneousStep from './MiscellaneousStep'
 import SequenceFileStep from './SequenceFileStep'
 import GeneListStep from './GeneListStep'
 import LoadingSpinner from '~/lib/LoadingSpinner'
+import H5adStep from './H5adStep'
```

Comment:
Blocker: As part of substantive architecture refactor for h5ad -> AnnData:

```suggestion
import AnnDataStep from './AnnDataStep'
```